docs/pages/router/reference/link-preview.mdx

```diff
@@ -0,0 +1,228 @@
+---
+title: Link Preview
+description: Learn how to add a preview to your link on iOS.
+hasVideoLink: false
+---
+
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+import { FileTree } from '~/ui/components/FileTree';
+
+<ContentSpotlight
+  alt="A video presenting the link preview feature in Expo Router."
+  file="/expo-router/link-preview.mp4"
+/>
+
+This guide will show you how to add and customize a link preview for your app on iOS. A link preview is a feature commonly used on iOS to show user a preview of the content linked by the link.
+If you have a link in your app, you can add preview by simply wrapping the link content with `<Link.Trigger>` component and adding `<Link.Preview>` component. This will create a preview of the page that the link points to.
+
+```tsx
+import { Link } from 'expo-router';
+
+export default function Page() {
+  return (
+    <Link href="/about">
+      <Link.Trigger>About</Link.Trigger>
+      <Link.Preview />
+    </Link>
+  );
+}
+```
+
+## New components
+
+### Link.Trigger
+
+A wrapper for the link's trigger. Anything that was previously passed as a child to `<Link>` should now be wrapped inside `<Link.Trigger>` when using `<Link.Preview>`.
```

Comment:
```suggestion
A wrapper for the link's displayed content. Anything that was previously passed as a child to `<Link>` should now be wrapped inside `<Link.Trigger>` when adding a link preview.
```

- nit: ~~trigger~~ -> displayed content. I believe this is more intuitive.
- Removed `Link.Preview` mention here, since it could be confusing without a code sample.